SimPEG/directives/directives.py

```diff
@@ -2984,11 +2988,10 @@ def save_components(self, iteration: int, values: list[np.ndarray] = None):
                             data, base_name
                         )
 
-    def save_log(self, iteration: int):
+    def write_update(self, iteration: int):
         """
-        Save iteration metrics to comments.
+        Write update to file.
         """
-
         dirpath = os.path.dirname(self._h5_file)
```

Comment:
We don't have to commit to it here, but i guess we should run through simpeg and update to pathlib at some point.

Reply:
100%. Doing it now.